examples/quant.py

```diff
@@ -0,0 +1,139 @@
+'''
```

Comment:
This benchmark is more of a use case for supporting boolean masks. It was ported from some C code we happened to have our hands on. The motivation for the original code was unclear to me at least. So if you also happen to think that it may not add much to Numba, I'm happy to remove it from this PR. What do you think? @ehsantn @DrTodd13

Reply:
Yes, I say remove it from the PR.